src/app/GitExtUtils/GitUI/Interops/User32/GetWindowLong.cs

```diff
@@ -0,0 +1,9 @@
+﻿using System.Runtime.InteropServices;
+
+namespace System;
+
+internal static partial class NativeMethods
+{
+    [DllImport("user32.dll")]
+    internal static extern int GetWindowLong(nint hwnd, int nIndex);
```

Comment:
I don't believe this is a correct import. On 64-bit systems, it should be `GetWindowLongPtrW`:
https://github.com/dotnet/winforms/blob/698cfb403515150b08ca8cd1f201b5d0aacde813/src/System.Windows.Forms.Primitives/src/Interop/User32/Interop.GetWindowLong.cs#L11-L26